dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -2439,7 +2156,7 @@ public void findRelationshipByLabelTest() throws Exception {
 
         // We're creating a Relationship of type isAuthorOfPublication between a publication and an author
         Relationship relationship3 = RelationshipBuilder
-            .createRelationshipBuilder(context, publication, author1, isAuthorOfPublicationRelationshipType).build();
+            .createRelationshipBuilder(context, publication1, author1, isAuthorOfPublicationRelationshipType).build();
```

Comment:
Looking at the file, I think you need a `context.restoreAuthSystemState();` call somewhere around here (it seems to be missing, if I'm not mistaken)?

Reply:
The restore has been included now